blocks/feature-layout/feature-layout.test.js

```diff
@@ -0,0 +1,23 @@
+describe('Feature-Layout Block', () => {
+  beforeAll(async () => {
+    await page.goto(`${global.BASE_URL}pattern-library/`);
+  });
+
+  it('should render the feature-layout block', async () => {
+    await page.waitForSelector('.feature-layout');
+    const featureLayouts = await page.$$('.feature-layout');
```

Comment:
I don't think you need the double awaits here, you can remove line 7? do the tests work if you do that?